tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
```

Comment:
Please add the element below to this property group.

```xml
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Reply:
Added element `AnalysisMode`